@utahdts/utah-design-system/react/enums/childrenMenuTypes.js

```diff
@@ -0,0 +1,9 @@
+/** @typedef {import('@utahdts/utah-design-system-header').childrenMenuType} ChildrenMenuTypes */
```

Comment:
Why rename the import? This could lead to confusion and an inability to find things later?